Gruntfile.js

```diff
@@ -46,8 +46,8 @@ module.exports = function(grunt) {
           file: 'test/fixtures/_component.json',
           changelog: 'test/fixtures/_CHANGELOG.md',
           additionalFiles: ['test/fixtures/_bower.json'],
-          changelogText: '### <%= version %>\n',
-          commitMessage: 'v<%= version %>'
+          changelogText: grunt.template.process('### <%= version %>\n', {data: {'version': '0.0.13'}}),
```

Comment:
Why? In the /test/fixtures/component.json file the version will be always 0.0.12.